plotly/offline/offline.py

```diff
@@ -90,6 +75,11 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly'):
                                 of the chart that will export the chart to
                                 Plotly Cloud or Plotly Enterprise
     link_text (default='Export to plot.ly') -- the text of export link
+    validate (default=True) -- validate that all of the keys in the figure
+                               are valid? omit if your version of plotly.js
+                               has become outdated with your version of
+                               graph_reference.json or if you need to include
+                               extra, unnecessary keys in your figure.
```

Comment:
Here's my rational: 

- It's easier for users to understand what's going on. Upgrading the python client is a natural action taken by users who want to new functionality. Upgrading `plotly.js` is separately is another step to teach and document. 
- We can formally introduce new functionality as new releases. A new chart type in `plotly.js` will trigger new chart type releases in Python, and update the changelog accordingly.
- We can test the integration of R and Python with `plotly.js` independently, and be confident about our releases (and documentation)
- It's easier for us to document. If a user views our docs, they can check their version of the python client against the documentation's versions. If something doesn't work with their chart, it's easier for them to rule-out it being an issue with an incompatible version.
- It's easier for IT depts to manage - version x of the package will work exactly like every other user's version x.